doc/guides/using-internal-errors.md

```diff
@@ -15,9 +15,9 @@ to be breaking changes. By providing a permanent identifier for a specific
 error, we reduce the need for userland code to inspect error messages.
 
 *Note*: Switching an existing error to use the `internal/errors` module must be
-considered a `semver-major` change. However, once using `internal/errors`,
-changes to `internal/errors` error messages will be handled as `semver-minor`
-or `semver-patch`.
+considered a `semver-major` change. In the near future, after the transition
+has been completed, error message changes will be handled as `semver-minor` or
```

Comment:
* `will hopefully`? (if we're optimistic)
* `might`? (if we're skeptic)
* or even `error message changes will be reevaluated, with the goal being to handle then as` (if we want to explain)